### PR TITLE
networkquality: add page

### LINKED ISSUE #7445: page request: networkQuality
[networkQuality](https://support.apple.com/en-gb/HT212313) is a native command-line tool shipped in macos Monterey.

### PR DESCRIPTION
**Pull request by mfrw**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

Close #7445

### CODE REVIEW

**Comment by mfrw** [comment]
> The [build](https://github.com/tldr-pages/tldr/actions/runs/1462938032) for this PR failed with the following error(s):
> 
> ```
> pages/osx/networkQuality.md:0: TLDR109 File name should be lowercase
> ```
> 
> Please fix the error(s) and push again.

@navarroaxel @sbrl What should we do incase the page that we are adding is `camel-case` ?
Should we add the page as lower-case only ?

**Comment by navarroaxel** [comment]
@mfrw yeah, due to the case-insensitive file system in Windows, and sometimes in macOS. All filenames should be in `lowercase`.

File: pages/osx/networkquality.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# networkQuality
+
+> Measure the network quality by connecting to the Internet.
```
Comment (navarroaxel) [inline pages/osx/networkquality.md:3]: I think we don't have a better link than this right now.
```suggestion
> Measure the network quality by connecting to the internet.
> More information: <https://support.apple.com/HT212313>.
```

File: pages/osx/networkquality.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Test the network quality for the default interface:
```
Comment (navarroaxel) [inline pages/osx/networkquality.md:5]: Do this command run a speed test always? :thinking: I think the "quality" indicator is the `Responsiveness` (RPM) indicator.

File: pages/osx/networkquality.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+- Test sequential upload and download instead of parallel:
+
+`networkQuality -s`
+
+- Test a specified network interface:
+
+`networkQuality -I {{en0}}`
+
+- Display a verbose test output:
```
Comment (navarroaxel) [inline pages/osx/networkquality.md:17]:
```suggestion
-  Test the network quality with verbose output:
```

File: pages/osx/networkquality.md (hunk 1 above)
Comment (mfrw) [reply pages/osx/networkquality.md]: Yep, I tried searching for the page -- could not find anything :(

File: pages/osx/networkquality.md (hunk 2 above)
Comment (mfrw) [reply pages/osx/networkquality.md:6]: I think in my opinion it always runs a speed test.

File: pages/osx/networkquality.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# networkQuality
+
+> Measure the network quality by connecting to the internet.
+> More information: <https://support.apple.com/HT212313>.
+
+- Test the network quality for the default interface:
+
+`networkQuality`
+
+- Test sequential upload and download instead of parallel:
```
Comment (marchersimon) [inline pages/osx/networkquality.md:10]:
```suggestion
- Tes tthe network quality, by sequentially instead of parallel uploading and downloading:
```

Comment (sbrl) [reply pages/osx/networkquality.md:10]: Typo, `Tes tthe` → `Test the` @marchersimon?

**Review by CleanMachine1 — APPROVED** [review]
After comment is addressed.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**